compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmValueGen.java

```diff
@@ -296,7 +288,6 @@ private void createInstantiateMethod(ClassWriter cw, BRecordType recordType,
             }
         }
 
```

Comment:
These newlines are not added by me. I think they are added by the original author to group the code lines logically to improve clarity since the method is long. Hence I think it is better to keep them and remove only extra blank lines.